src/net.h

```diff
@@ -703,21 +703,36 @@ class V1TransportDeserializer final : public TransportDeserializer
     CNetMessage GetMessage(const CMessageHeader::MessageStartChars& message_start, int64_t time) override;
 };
 
+// ChaCha20 must never reuse a {key, nonce} for encryption nor may it be
+// used to encrypt more than 2^70 bytes under the same {key, nonce}
+// Re-key after 1GB (RFC4253 / SSH recommendation) or after 1h
+static constexpr unsigned int REKEY_LIMIT_BYTES = (1024 * 1024 * 1024);
+static constexpr unsigned int REKEY_LIMIT_TIME = 3600;
+static constexpr unsigned int REKEY_ABORT_LIMIT_BYTES = REKEY_LIMIT_BYTES * 1.1; // abort after ~10% tolerance buffer
+static constexpr unsigned int REKEY_ABORT_LIMIT_TIME = REKEY_LIMIT_BYTES * 1.1;  // abort after ~10% tolerance buffer
```

Comment:
You should precise in BIP that a rekey initiator should be conservative but rekey responder liberal in its window acceptance.

Reply:
Yes. That could help.